icons/flame-kindling.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M15.5 7c-1-1-2.5-2-3.5-5-1 0-4 3-1.6 5.6a2 2 0 1 1-3.3 2c-.1.3-.1.6-.1.9A4.95 4.95 0 1 0 15.5 7" />
```

Comment:
Looks pretty neat at 2px, but the bottom part is off grid and appears quite blurry at 1x.
```suggestion
  <path d="M15.5 6.5C14.5 5.5 13 5 12 2c-1 0-4 2.5-1.6 5.1a2 2 0 1 1-3.3 2c-.1.3-.1.6-.1.9a5 5 0 1 0 8.5-3.5" />
```
![image](https://github.com/lucide-icons/lucide/assets/17746067/852be131-d040-439c-af1f-f05c6511e755)

Reply:
@karsa-mistmere Ah yeah, it got over-optimised by https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit

Reply:
Not sure if this should be fixed by 42fbffb, but it still looks the same for me:
![image](https://github.com/lucide-icons/lucide/assets/17746067/b2dd98d4-5137-4a6e-a433-08711182d9ca)